ivy/functional/frontends/torch/reduction_ops.py

```diff
@@ -160,6 +160,7 @@ def median(input, dim=None, keepdim=False, *, out=None):
     return result
 
 
+@with_unsupported_dtypes({"1.26.1 and below": ("complex",)}, "numpy")
```

Comment:
could you please specify this unsupported dtype in the `numpy` backend implementation of `ivy.mean` as it's a backend-specific unsupported dtype? Thanks @yopknopixx 😄